backend/api/Context/InitDb.cs

```diff
@@ -0,0 +1,41 @@
+using api.Models;
+
+namespace api.Context;
+
+public static class InitDb
+{
+    public static readonly List<Robot> Robots = GetRobots();
+
+    private static List<Robot> GetRobots()
+    {
+        Robot robot1 = new Robot
+        {
+            Name = "Robot1",
+            Model = "Model1",
+            SerialNumber = "123",
+            Status = RobotStatus.Available
+        };
+
+        Robot robot2 = new Robot
+        {
+            Name = "Robot2",
+            Model = "Model2",
+            SerialNumber = "456",
+            Status = RobotStatus.Busy
+        };
+
+        return new List<Robot>(new Robot[] { robot1, robot2 });
+    }
+
+    public static void PopulateDb(FlotillaDbContext context)
+    {
+        if (context == null)
+        {
+            throw new ArgumentNullException(nameof(context));
+        }
```

Comment:
This is not needed because of the general nullable context feature.
If this method is attempted to be called with context=null, the compiler will error

Reply:
Agreed, fixed